src/modules/caseNotes/components/EnrollmentCaseNotes.tsx

```diff
@@ -119,15 +126,16 @@ const EnrollmentCaseNotes = () => {
         title='Case Notes'
         headerVariant='border'
         actions={
-          canEdit ? (
+          canEdit &&
+          !caseNotesFeature?.legacy && (
```

Comment:
The assumption is that `caseNotesFeature` is present, because this component is being rendered, right?
Maybe we should throw in a `... || !caseNotesFeatures) return <NotFound />;` to make that a little clearer.

Then at this point, we know it is present, so this can be updated to `!caseNotesFeature.legacy` which is easier to understand